lib/core/helpers/getOrCreateBridge.js

```diff
@@ -0,0 +1,40 @@
+var Q = require('q');
+
+/**
+ * Exports logic for creating a new bridge or reusing an existing bridge.
+ * @param {Object} opts - Contains the data for the client and the bridge name.
```

Comment:
Document the allowed options using a similar syntax as an object return except using multiple @param for each sub item of opts.

```JavaScript
/**
 * Documentation...
 * @param {Object} opts - ...
 * @param {Client} opts.client - ...
**/
```